runtime/gc_glue_java/ArrayletObjectModelBase.cpp

```diff
@@ -74,6 +74,15 @@ GC_ArrayletObjectModelBase::expandArrayletSubSpaceRange(MM_MemorySubSpace * subS
 	}
 }
 
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+void
+GC_ArrayletObjectModelBase::setEnableDoubleMapping(bool enableDoubleMapping)
+{
+	MM_GCExtensionsBase* extensions = MM_GCExtensionsBase::getExtensions(_omrVM);
+	extensions->isDoubleMapRequested = enableDoubleMapping;
+}
+#endif /* J9VM_GC_ENABLE_DOUBLE_MAP */
+
```

Comment:
no really need to for this wrapper. command line option parser can change isDoubleMapRequested  directly